### DISCUSSION TITLE
Once installed from the .deb release, where are the binaries?

### DISCUSSION THREAD

**Discussion by qathy** [google/orbit#4858] [Q&A] [answered]
Hello,
Sorry for this dumb question, but as I struggled quite a lot (and failed) to build Orbit, I've installed it from orbit_1.86dev-478-g3493933c5-1.jammy1_amd64.deb file obtained in the last release (daily build) available today.
Unfortunately, I don't find the binaries to launch, neither the GUI nor the service.
What am I doing wrong?

**Comment by beckerhe** [comment, marked as answer]
Hi @qathy,
the files should be under `/opt/orbitprofiler`.

`/opt/orbitprofiler/Orbit` is the UI and `/opt/orbitprofiler/OrbitService` is the service.

I hope that helps.

- Henning

**Reply by qathy** [reply]
Thanks!
